R/spec.R

```diff
@@ -21,12 +21,14 @@ spec <- function(spec, env = caller_env(), signaller = "signal_lifecycle") {
 }
 
 spec_what <- function(what, arg, signaller) {
-  if (!is_string(what)) {
-    lifecycle_abort("`what` must be a string")
+  if (is_call(what)) {
+    call <- what
+  } else if (is_string(what)) {
+    call <- parse_expr(what)
```

Comment:
Do we dare soft-deprecating this? 🙃

Reply:
It's tempting, but I think it will cause more hassle than its worth.